packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.stories.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Grommet } from 'grommet'
+import { withKnobs, boolean, text, object } from '@storybook/addon-knobs'
+import VariableStarViewer from './VariableStarViewer'
+import ZoomInButton from '../../../ImageToolbar/components/ZoomInButton/ZoomInButton'
+import ZoomOutButton from '../../../ImageToolbar/components/ZoomOutButton/ZoomOutButton'
+import ResetButton from '../../../ImageToolbar/components/ResetButton/ResetButton'
+import {
+  variableStarAmplitudeMockData,
+  variableStarPeriodMockData
+} from '../BarChartViewer/mockData'
+import variableStar from '../../helpers/mockLightCurves/variableStar'
+import readme from './README.md'
+import backgrounds from '../../../../../../../.storybook/lib/backgrounds'
+import image from './mocks/temperature.png'
+
+const config = {
+  notes: {
+    markdown: readme
+  }
+}
+
+let zoomCallback
+
+function onZoom(type) {
+  zoomCallback(type)
+}
+
+function setZoomCallback(callback) {
+  zoomCallback = callback
+}
+
+const stories = storiesOf('Subject Viewers | VariableStarViewer', module)
+
+stories.addDecorator(withKnobs)
+// stories.addParameters({ viewport: { defaultViewport: 'responsive' } })
```

Comment:
Yep. There's a warning being thrown from the add-on probably since I updated storybook, so I need to look into what needs to be fixed with the configuration. I commented it out to get rid of the warning for now.